FridgeExercise/Program.cs

```diff
@@ -1,2 +1,242 @@
+						fridge.PutItemInTheFridge(itemToInsert, fridge.Shelves);
+					}
+					else
+						Console.WriteLine("There is no such a type:" + type);
+				}
+				else
+					Console.WriteLine("There is no such a kosherType: " + kosherType);
+
+			}
+
+		}
+
+		static void Main(string[] args)
+		{
+			var ThisProgramObject = new Program();
+			List<Refrigerator> allRefrigerators = new List<Refrigerator>();
+
+			Refrigerator refrigerator2 = new Refrigerator("Sharp", "write", 6);
+			allRefrigerators.Add(refrigerator2);
+			refrigerator2.Shelves.Add(new Shelf(1, 30));
+			refrigerator2.Shelves.Add(new Shelf(2, 30));
+			refrigerator2.Shelves.Add(new Shelf(3, 30));
+			refrigerator2.Shelves.Add(new Shelf(4, 25));
+			refrigerator2.Shelves.Add(new Shelf(3, 10));
+			refrigerator2.Shelves.Add(new Shelf(4, 17.5));
+			Item item21 = new Item("milk", 2023, 10, 15, 20, "Dairy", "Drink");
+			Item item22 = new Item("chily", 2023, 10, 22, 10, "Parve", "Drink");
+			Item item23 = new Item("meat", 2023, 10, 17, 30, "Meat", "Food");
+			Item item24 = new Item("chocolate", 2023, 10, 20, 5, "Dairy", "Drink");
+			refrigerator2.PutItemInTheFridge(item21, refrigerator2.Shelves);
+			refrigerator2.PutItemInTheFridge(item22, refrigerator2.Shelves);
+			refrigerator2.PutItemInTheFridge(item23, refrigerator2.Shelves);
+			refrigerator2.PutItemInTheFridge(item24, refrigerator2.Shelves);
+
+
+
+			Refrigerator refrigerator=new Refrigerator("samsung","gray",4);
+			allRefrigerators.Add(refrigerator);
+			refrigerator.Shelves.Add(new Shelf(1, 10));
+			refrigerator.Shelves.Add(new Shelf(2, 10));
+			refrigerator.Shelves.Add(new Shelf(3, 10));
+			refrigerator.Shelves.Add(new Shelf(4, 10));
+			//refrigerator.PutItemInTheFridge(new Item("milk2", 2023, 10, 30, 2, "Dairy", "Drink"), refrigerator.Shelves);
+			//refrigerator.PutItemInTheFridge(new Item("chily0", 2023, 10, 22, 4, "Parve", "Drink"), refrigerator.Shelves);
+			////refrigerator.PutItemInTheFridge(new Item("meat0", 2023, 10, 17, 10, "Meat", "Food"), refrigerator.Shelves);
+			////refrigerator.PutItemInTheFridge(new Item("chocolate", 2024, 10, 20, 5, "Dairy", "Drink"), refrigerator.Shelves);
+			//refrigerator.PutItemInTheFridge(new Item("milk", 2023, 10, 25, 10, "Dairy", "Drink"), refrigerator.Shelves);
+			////refrigerator.PutItemInTheFridge(new Item("meat", 2023, 10, 27, 15, "Meat", "Drink"), refrigerator.Shelves);
+			//refrigerator.PutItemInTheFridge(new Item("meat2", 2023, 10, 31, 2, "Meat", "Drink"), refrigerator.Shelves);
+			//refrigerator.PutItemInTheFridge(new Item("chily1", 2023, 10, 24, 10, "Parve", "Drink"), refrigerator.Shelves);
+			////refrigerator.PutItemInTheFridge(new Item("milk0", 2023, 10, 20, 19, "Dairy", "Drink"), refrigerator.Shelves);
+			//refrigerator.PutItemInTheFridge(new Item("chily2", 2023, 10, 30, 10, "Parve", "Drink"), refrigerator.Shelves);
+			//refrigerator.PutItemInTheFridge(new Item("meat3", 2023, 11, 01, 1, "Meat", "Drink"), refrigerator.Shelves);
+
+
+			int action =0;
```

Comment:
```suggestion
			int action = 0;
```

I also see you're resetting this value each loop. Since you're skipping the loop if the user input is not valid and you're not using the variable outside the context of the loop, you can simply initialize it within the loop.